src/notebook/widget.ts

```diff
@@ -337,6 +337,25 @@ class NotebookWidget extends Widget {
 }
 
 
+function createElements(tag: string, attr?: any, children?: HTMLElement[] | string): HTMLElement {
```

Comment:
Actually, we're losing type safety and clarity by doing this.  I think this should be an interface:

```typescript
interface IElementOptions {
   tag?: string;
   className?: string;
   children?: HTMLElement[];
   title?: string;
   value?: string;
}
```

We could add more options as desired.